src/components/common/LabeledSelect.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from 'styled-components'
+import Select from 'react-select'
+import find from "lodash/find"
+import map from "lodash/map"
+
+const getStringValues = (values=[], isMulti="false") => (
```

Comment:
isMulti="false"? we're passing strings for booleans now?

Reply:
:)))) nope...